README.md

```diff
@@ -41,21 +41,13 @@ $ pip install --process-dependency-links 'git+https://github.com/GrafeasGroup/to
 Monitoring daemon (via Redis queue):
 
 - Pull job (by post id) off of queue:
-  - Download image
-  - OCR the image
+  - Get image URL
+  - Use orc.space API on the image
   - If OCR successful:
     - Post OCR-ed content to post on /r/TranscribersOfReddit in 9000 character chunks, replying to previous comment when [over 9000][over-9000] characters
-  - Delete local copy of image
 
 [over-9000]: https://tenor.com/view/dragonball-z-super-saiyan-charging-yelling-gif-4987448
 
-### Running Apprentice Bot
-
-```
-$ tor-apprentice
-# => [daemon mode + logging]
-```
```

Comment:
I'd argue that we should leave in the usage docs, even if it is only us who plans to use it. Usage is one of the key points of a good README, in my opinion, and the main point of the documentation.

Reply:
~~I guess I don't understand—where do we use this `tor-apprentice` command?~~

I just realized that's the script it's installed too. It might be a good idea to rename this in a different PR.